test/Interop/Cxx/namespace/class-inline-namespace-irgen.swift

```diff
@@ -0,0 +1,100 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: %target-swift-emit-ir -I %t/Inputs -enable-cxx-interop %t/test.swift | %FileCheck %t/test.swift
+
+
+//--- Inputs/module.modulemap
+module std {
+  header "string.h"
```

Comment:
`string.h` is a header that may already be in our search paths, so maybe another name is better.